.circleci/config.yml

```diff
@@ -0,0 +1,196 @@
+            python3 -V
+      - run:
+          name: Docker-Compose
+          command: cd $DIRECTORY && docker-compose up -d
+
+  send_coverage:
+    steps:
+      - run:
+          name: send coverage
+          command: bash <(curl -s https://codecov.io/bash) -F ${DIRECTORY//-} || echo "Codecov did not collect coverage reports"
+
+  install_dockerize:
+    steps:
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.6.1
+
+  wait_for_db:
+    steps:
+      - run:
+          name: Wait for db
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+
+  wait_for_elastic_search:
+    steps:
+      - run:
+          name: Wait for ES
+          command: |
+            wget --output-document /dev/null --waitretry=5 --tries=10 --retry-connrefused localhost:9200 || true
+            wget --output-document /dev/null --waitretry=5 --tries=10 --retry-connrefused localhost:9200 || true
+
+  run_git_secrets_scan:
+    steps:
+      - run:
+          name: Run git-secrets scan on dockstore repository
+          # Needs to be after a build so that the correct configuration is set
+          # by the exec-maven plugin
+          command: |
+            git secrets --scan
+
+  install-git-secrets:
+    steps:
+      - run:
+          name: Install git-secrets
+          command: |
+            wget --no-verbose -O git-secrets-1.3.0.tar.gz https://github.com/awslabs/git-secrets/archive/1.3.0.tar.gz
+            tar -zxf git-secrets-1.3.0.tar.gz
+            cd git-secrets-1.3.0
+            sudo make install
+
+  setup_postgres:
+    steps:
+      - run:
+          name: setup postgres user and DB
+          command: |
+            psql -c "create user dockstore with password 'dockstore' createdb;" -U postgres
+            psql -c 'create database webservice_test with owner = dockstore;' -U postgres
+            psql -c "ALTER USER dockstore WITH superuser;" -U postgres
+
+  install_postgres_sql_client:
+    steps:
+      - run:
+          name: Install postgresql client
+          command: |
+            sudo rm -rf /var/lib/apt/lists/*
+            # heroku.com is not updating their keys
+            # https://github.com/heroku/cli/issues/1464
+            curl https://cli-assets.heroku.com/apt/release.key | sudo apt-key add -
+            sudo apt update
+            # NOTE: this may be installing latest client; perhaps should
+            # specify version
+            sudo apt install -y postgresql-client
+
+  setup_for_integration_tests:
+    steps:
+      - get_workspace_and_dependencies
+      - setup_machine
+      - setup_for_tests
+      - wait_for_elastic_search
+
+  get_workspace_and_dependencies:
```

Comment:
Think we need a CircleCI equivalent of https://ucsc-cgl.atlassian.net/browse/SEAB-4258
Create a ticket and link https://discuss.circleci.com/t/feature-request-share-config-yml-between-projects/24287
https://circleci.com/docs/reusing-config/

May be a co-op project for the next term

Reply:
Created https://ucsc-cgl.atlassian.net/browse/SEAB-5299